pvlib/tests/test_modelchain.py

```diff
@@ -481,6 +489,14 @@ def test_ac_model_user_func(pvwatts_dc_pvwatts_ac_system, location, weather,
     assert not mc.ac.empty
 
 
+def test_ac_model_not_a_model(pvwatts_dc_pvwatts_ac_system, location, weather):
+    exc_text = 'not a valid AC power model'
+    with pytest.raises(ValueError, match=exc_text):
+        ModelChain(pvwatts_dc_pvwatts_ac_system, location,
+                   ac_model='not_a_model', aoi_model='no_loss',
+                   spectral_model='no_loss')
+
+
```

Comment:
Add a test that will fail on pvlib 0.9 if the old ac method names are provided.

Reply:
@wholmgren PTAL not sure I got the test right

https://github.com/pvlib/pvlib-python/blob/7663321a3936820265b89c6c88a7c0ee60e263fa/pvlib/tests/test_modelchain.py#L716